tapdb/sqlc/migrations/000026_asset_group_version_customsubtree.up.sql

```diff
@@ -0,0 +1,31 @@
+-- Add version field to table asset_groups.
+ALTER TABLE asset_groups ADD COLUMN version INTEGER NOT NULL DEFAULT 0;
+
+-- Add custom_subtree_root field to the table asset_groups. This optional field
+-- is used to store the root of the custom subtree for the asset group. The
+-- custom subtree, if provided, represents a subtree of the final tapscript
+-- tree.
+ALTER TABLE asset_groups ADD COLUMN custom_subtree_root BLOB;
```

Comment:
Why isn't the existing field sufficient? 

https://github.com/lightninglabs/taproot-assets/blob/a00fd22d4e009dc83cec5a093d91e0db1f65ef50/tapdb/sqlc/migrations/000002_assets.up.sql#L95

Reply:
I think we only need the version here added for this migration.

Reply:
We need to maintain the custom subtree root separately from the tapscript root because this table will later be used to construct the mint proof group key reveal, which requires specifying the custom subtree root.

Reply:
We need the `version` column because the group key reveal formalisation might differ in the future even if we still have a separate custom subtree root.